docusaurus.config.js

```diff
@@ -4,9 +4,9 @@ module.exports = {
   title: "Camunda Platform 8 Docs",
   tagline: "Documentation for all components of Camunda Platform 8",
   // url: "https://camunda-cloud.github.io",
-  url: "https://docs.camunda.io",
+  url: "https://unsupported.docs.camunda.io",
   // baseUrl: "/camunda-cloud-documentation/",
-  baseUrl: "/",
+  baseUrl: "/1.3/",
```

Comment:
When running locally, this will serve everything at `localhost:3000/1.3/docs/....`. When deployed, it will be `unsupported.docs.camunda.io/1.3/docs/....`. 

Changing this property to reflect this enables docusaurus to generate all links correctly within the docs.